bin/common.sh

```diff
@@ -49,7 +50,7 @@ function apply_manifests() {
     wait_for_operators_to_be_running || return $?
   done
 
-  scale_deployment "kafka-broker-dispatcher" 3 || return $?
+  scale_deployment "kafka-broker-dispatcher" 5 || return $?
```

Comment:
No need to address: but interested in the resasoning behind those numbers (can be done later).

same for the `machineset` scaling above

Reply:
This dispatcher and receiver scaling probably predates the SRVKE-1243 fix, so I'll re-evaluate those numbers

for machine set, when we have many triggers we request at least 500m CPU for each subscriber and each machine has 16cores, so that quickly requires many cores, this as well predates the use of sacura which might be a little more light in terms of CPU, so I'll probably retry with less requested CPUs in the future.